python/lsst/atmospec/nightAnalysis.py

```diff
@@ -165,27 +165,41 @@ def removeSeqNums(self, seqNums):
                 self.seqNums.remove(seqNum)
                 self.data.pop(seqNum)
 
+    def _getExposureRecords(self):
+        """Retrieve the exposure records for the relevant exposures.
+
+        Returned in same order as ``self.seqNums``.
```

Comment:
These should now be sorted, so this comment can be removed.

Reply:
I'm a bit worried that that ordering is an implicit assumption but is not enforced anywhere. If someone decided to rearrange some code and the order changed after the original sorting would people be more confused or less confused if this always returned an order that matches `self.seqNums` vs always returning sorted by sequence number?

Reply:
Uggg, well, the whole thing ideally would return the dict that you neatly construct and then discard, but that would likely mean more changes elsewhere, as you'd be keying off the dict rather than just iterating. If you make it return the dict and leave it broken I'm happy to fix this up to use it like that if/when I ever use this code again - I'm certain nobody else is using it right now.

Reply:
Changed to return the dict and the callers now get to decide on the order they want to use.